__mocks__/@thecointech/market-status.ts

```diff
@@ -0,0 +1,20 @@
+
+
+import { DateTime } from 'luxon';
```

Comment:
Mocked version still limits open timestamps to open hours (9-4:30 weekdays only)